### DISCUSSION TITLE
Flow Information section blank

### DISCUSSION THREAD

**Discussion by KarciSF** [jongpie/NebulaLogger#623] [Q&A] [answered]
Hey guys,

I need your help. My log entries are missing the entire 'Flow Information' section. I've checked the configuration, and `QueryFlowDefinitionViewData` is set to true. However, I can't find any fields labeled as 'flow' in the `LogEntryEvent__e` object. Am I overlooking something, or is this a known issue?

<img width="1499" alt="Screenshot 2024-02-05 at 11 37 20" src="https://github.com/jongpie/NebulaLogger/assets/90701851/5a0dce48-548d-47c4-b4f0-33d2bee9a17a">

**Comment by jongpie** [comment]
Hi @KarciSF - the 'Flow Information' section is populated based on the 'Flow API Name' you provide when configuring the logging invocable action in Flow. Based on your screenshot of the `LogEntry__c` (in the 'Origin' field), it looks like the provided 'Flow API Name' value is 'Sabalaba'. Can you confirm if that value matches the actual API name of your Flow?

And just for clarification, these Flow fields only exist on the `LogEntry__c` custom object - the `LogEntryEvent__e` platform event does not have any Flow-related fields.

**Reply by KarciSF** [reply]
Hey @jongpie. Quite a strange name for playing around, right? But I can confirm that 'Sabalaba' is indeed the correct name. I've just figured out why it wasn't populated. This flow doesn't have any active version, so I assume the query for FlowDefinitionView is looking for an active version.

So, I can confirm it's now working :)  I appreciate your response on this. 

I have another question, but I'll post it as a separate question.

**Reply by jongpie** [reply]
@KarciSF  that's a good point, and you are correct - the query for `FlowDefinitionView` is only returning active Flows. Just for my own understanding/clarification - when you generated the log entries, were you debugging (in Flow Builder) an inactive Flow?

**Reply by KarciSF** [reply]
Yes, exactly just playing around testing the different actions and their behaviour and appearance of the logs in the system.

**Reply by jongpie** [reply, marked as answer]
That makes sense, thanks for clarifying! I think the scenario is difficult to log accurately - as far as I'm aware, there's not a way for Nebula Logger to know that you're debugging an inactive Flow in Flow Builder, so the "safer" option is to only query active `FlowDefinitionView` records. I'll try to eventually revisit this to see if there's a way to detect that the Flow is running in debug mode, but this might be a Salesforce limitation.

**Reply by KarciSF** [reply]
Thank you for explaining that! It turns out it was just a gap in my knowledge 😄. Everything else is running smoothly in debug mode, so missing this bit of information isn't critical while I'm playing arounc in debug mode. Thanks again!

**Reply by jongpie** [reply]
No problem! I've marked [my last comment](https://github.com/jongpie/NebulaLogger/discussions/623#discussioncomment-8384921) as the answer for this discussion, but let me know if you have any other follow-up questions about this.
